python/lsst/ts/GenericCamera/driver/genericcamera.py

```diff
@@ -35,6 +56,14 @@ def __init__(self, log=None):
         else:
             self.log = log
 
+        # Load the generic FITS header items.
+        fhig = FitsHeaderItemsGenerator()
```

Comment:
Se my comment above. Either do this in one line or make a more readable name for the variable.